configs/.eslintrc.js

```diff
@@ -13,6 +13,12 @@ module.exports = {
 		ecmaVersion: 9
 	},
 	rules: {
+		'max-len': [
+			"error",
+			{
+				"ignorePattern": "^require |^import |^export {(.*?)}|`.*`|'.*'|// .*"
```

Comment:
why not simply `.*$`?

Reply:
after the begin of line match ofc

Reply:
No particular reason. The example given bij tslint listed it as such. It isn't really necessary because ignorePatterns only work for single line. `.*` is thus already until the end of line.